### DISCUSSION TITLE
Please Share Your Plugin with Umami

### DISCUSSION THREAD

**Discussion by mdestafadilah** [umami-software/umami#879] [Ideas]
Hi, First. I fall in love with Umami, it's easy interface and useful for my several Project.

I think we must collecting  some Plugin/ or Tools whatever name, have done with Umami, for example, I see, in Gatsby JS someone create plugin, install Umami to Gatsby Project, you can check [here](https://www.npmjs.com/package/gatsby-plugin-umami) , may we start from little to awesome tools like Umami do.

Thanks You!

**Comment by mikecao** [comment]
They are listed here, https://umami.is/docs/useful-links
